src/Tests/Microsoft.NET.Publish.Tests/PublishDepsFilePathTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using FluentAssertions;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Publish.Tests
+{
+    public class PublishDepsFilePathTests : SdkTest
+    {
+        public PublishDepsFilePathTests(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Fact]
+        public void PublishDepsFilePathIsSetAsExpectedForNormalApps()
+        {
+            var testProject = SetupProject(singleFile: false);
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+            var restoreCommand = new RestoreCommand(Log, testAsset.Path, testProject.Name);
+            restoreCommand
+                .Execute()
+                .Should()
+                .Pass();
+
+            var projectPath = Path.Combine(testAsset.Path, testProject.Name);
+            var targetFramework = testProject.TargetFrameworks;
+
+            var publishDir = GetPropertyValue(projectPath, targetFramework, "PublishDir");
+            var projectDepsFileName = GetPropertyValue(projectPath, targetFramework, "ProjectDepsFileName");
+            var publishDepsFilePath = GetPropertyValue(projectPath, targetFramework, "PublishDepsFilePath");
+            var expectedDepsFilePath = $"{publishDir}{projectDepsFileName}";
+
+            publishDepsFilePath.Equals(expectedDepsFilePath).Should().BeTrue();
```

Comment:
I think it's better to write the assertion like this:

```c#
publishDepsFilePath.Should().Be(expectedDepsFilePath)
```

That gives more diagnosable failure messages.

(This also applies to the other assertions)